go.mod

```diff
@@ -28,7 +28,7 @@ require (
 	github.com/golang/mock v1.6.0
 	github.com/gorilla/mux v1.8.1
 	github.com/grafana/otel-profiling-go v0.5.1
-	github.com/grafana/pyroscope-go v1.1.1
```

Comment:
Old pyro version lacked support for 1.23:
https://github.com/grafana/pyroscope-go/issues/118